server/master/src/main/java/org/apache/accumulo/master/recovery/RecoveryManager.java

```diff
@@ -132,6 +148,19 @@ private void initiateSort(String sortId, String source, final String destination
     log.info("Created zookeeper entry " + path + " with data " + work);
   }
 
+  private boolean exists(final Path path) throws IOException {
+    try {
+      return existenceCache.get(path, new Callable<Boolean>() {
+        @Override
+        public Boolean call() throws Exception {
+          return master.getFileSystem().exists(path);
+        }
+      });
```

Comment:
Callable is also a functional interface:

```java
return existenceCache.get(path, () -> master.getFileSystem().exists(path));
```

Reply:
Its 1.9, so no lambda.